tests/core/arch-arm64/stalker-arm64.c

```diff
@@ -1240,6 +1241,33 @@ TESTCASE (follow_thread)
 #endif
 }
 
+static gpointer
+_inc_integer(gpointer number)
```

Comment:
```suggestion
increment_integer (gpointer data)
```

Reply:
Should move this below the function (newspaper code ordering – read from the top, with details further down).